looprpc/client.proto

```diff
@@ -635,6 +635,11 @@ message InQuoteResponse {
     On-chain cltv expiry delta
     */
     int32 cltv_delta = 5;
+
+    /*
+    The confirmation target to be used for the sweep of the on-chain HTLC.
```

Comment:
For loop-in this conf target is used for the initial htlc, because the client publishes on-chain, so perhaps "used to publish the on-chain htlc" over "sweep"?

Reply:
Got it, now changed.